### ISSUE TITLE
[BUG] Unable to import from Keystore file as wallet disappears

### ISSUE THREAD

**Issue by kf106** [wavesplatform/waves-keeper#192]
**Describe the bug**
If you go to the create a new account pane and choose import keystore file, when you click on Browse, although a file select dialog appears, the wallet pane disappears. On returning to the wallet the selected file is not remembered.

As a result it is not possible to load a keystore file.

**To Reproduce**

1. Go to create account
2. Click on Keystore File
3. Select a file
4. Wallet pane disappears and you can't continue

**Expected behavior**
Wallet should stay visible during file select.

**Screenshots**
N/A

**Environment**
Please complete the following information.

Waves Keeper 2.0.0 zip download
Linux 5.11.0-43-generic #47~20.04.2-Ubuntu SMP Mon Dec 13 11:06:56 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
Brave V1.33.106 and Chrome Version 96.0.4664.110 (Official Build) (64-bit)

**Comment by kf106** [comment]
Incidentally, if you have inspect > console open then the wallet doesn't disappear and you can load  the keystore file. But general users won't be doing that.

**Comment by msmolyakov** [comment]
Yes, in some browsers the file selection window causes the popup to close. We will fix it in the next weeks
